src/adapter-types/src/dyncfgs.rs

```diff
@@ -21,14 +21,14 @@ pub const ALLOW_USER_SESSIONS: Config<bool> = Config::new(
 
 pub const ENABLE_0DT_DEPLOYMENT: Config<bool> = Config::new(
     "enable_0dt_deployment",
-    false,
+    true,
```

Comment:
SGTM, but does this cause any problems for restart tests?

Reply:
We already explicitly set this to true/false in all of our tests instead of relying on the default value, so no concern.